.github/workflows/ci.yml

```diff
@@ -37,21 +35,48 @@ jobs:
       - name: Build && install up-cpp
         shell: bash
         run: |
-          mkdir -p build && cd build
-          conan install .. -o build_testing=True
-          cmake -S .. -DCMAKE_TOOLCHAIN_FILE=conan_toolchain.cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_PREFIX=~/local
-          cmake --build . --target install -- -j
+          conan install . --build=missing -o build_testing=True
 
       - name: Run the test
         shell: bash
         run: |
-          cd build
+          cd build/Release
           ctest
 
       - name: Create up-cpp Conan package
         shell: bash
         run: |
-          conan create . --build=missing
+          cd conan-recipes
+          conan create .. --build=missing
+
+  package:
```

Comment:
@stevenhartley, @gregmedd -- FYI I updated to separate out the test vs package flows to distinguish between them and their conanfile.py